R/KYCG.R

```diff
@@ -225,7 +226,9 @@ testEnrichmentFisher <- function(query, database, universe) {
         test = "Log2FC",
         nQ = length(query),
         nD = length(database),
-        overlap = q_and_d)
+        overlap = q_and_d,
+        CpGs = I(list(q_and_d_probes)))
```

Comment:
I still think It might be cumbersome to have the result always contain the list of overlapping CpGs. In the spirit of Occam's razor, can't we make do with something like
```{r}
intersect(query, KYCG_getDBs("MM285.chromHMM", db_names="TssA"))
```
without touching the testEnrichment function?

Reply:
Yes agreed. For me personally while testing it would be easier to just grab the CpGs from the results data frame instead of going back and plugging in the db_name and intersecting. But it probably doesn't warrant the extra data in there

Reply:
So should I make a separate function to get overlapping CpGs using your code above, or do we just leave it to the user to find the intersection if they are interested in exploring a particular set more in depth?

Reply:
Since this is just one line, my preference is to just leave it to the user. We can add that example in the vignette so we know it is purposeful (to leave out the CG list in the result data frame). Hope that makes sense.